com.github.micahflee.torbrowser-launcher.yaml

```diff
@@ -0,0 +1,120 @@
+app-id: com.github.micahflee.torbrowser-launcher
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+separate-locales: false
+command: torbrowser-launcher
+rename-desktop-file: torbrowser.desktop
+rename-appdata-file: torbrowser.appdata.xml
+rename-icon: torbrowser
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --persist=.config/torbrowser
+  - --persist=.local/share/torbrowser
+  - --filesystem=xdg-download
+modules:
+  - shared-modules/dbus-glib/dbus-glib-0.110.json
+
+  - name: python3-sip
+    buildsystem: simple
+    build-commands:
+      - python3 configure.py
```

Comment:
Hm, I'd rather rewrite it as a native autotools module, e.g. add a `shell` source with command `ln -s configure.py configure` and use `config-args`. But it's up to you.

Reply:
I will probably keep the current form for now. But thanks for the tip!

 Anyway, feel free to send a PR if you want, I will happily merge it. :-)